app/views/todos/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<ul>
+  <% @incomplete_todos.each do |todo| %>
+    <li>
+      <span><%= link_to todo.title, todo_path(todo, todo: {completed: true}), method: :patch %></span>
+      <span><%= todo.due_date %></span>
+      <span><%= link_to "show", todo_path(todo) %></span>
+      <span><%= link_to "edit", edit_todo_path(todo) %></span>
+      <span><%= link_to "delete", todo_path(todo), method: :delete %></span>
+    </li>
+  <% end %>
+
+  <% @complete_todos.each do |todo| %>
+    <li class="completed">
+      <span><%= link_to todo.title, todo_path(todo, todo: {completed: false}), method: :patch %></span>
+      <span><%= todo.due_date %></span>
+      <span><%= link_to "show", todo_path(todo) %></span>
+      <span><%= link_to "edit", edit_todo_path(todo) %></span>
+      <span><%= link_to "delete", todo_path(todo), method: :delete %></span>
+    </li>
```

Comment:
Seeing as how you use the same code in the `completed` todos loop as the `incomplete` todos loop, you could extract it to a partial.